FirebaseFunctionsSwift/Sources/FunctionsError.swift

```diff
@@ -14,20 +14,20 @@
 
 import Foundation
 
-// The error domain for codes in the FIRFunctionsErrorCode enum.
-public let FunctionsErrorDomain: String = "com.firebase.functions"
```

Comment:
Let's keep the Swift version the same, and add a double prefix to the Swift version of the ObjC visible error